opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/models/Televisions.java

```diff
@@ -0,0 +1,191 @@
+package nl.novi.opdrachttechiteasy.models;
+import jakarta.persistence.*;
+
+@Entity
+@Table
+public class Televisions {
```

Comment:
We willen de class "Television" noemen en de tabel in de DB "televisions". Dat doen we door `@Table(name = "televisions")`